packages/jupytercad-opencascade/lib/index.js

```diff
@@ -0,0 +1,9 @@
+import initOpenCascade from 'opencascade.js';
```

Comment:
Try to get rid of these as typings are not forwarded properly and not visible in VSCode